common/culture/cultures/orc.txt

```diff
@@ -912,7 +912,7 @@
 		mother_name_chance = 5
```

Comment:
> The ethnicity was created by making an "average" of ogre and orc genes, so even if they don't look exactly like the Mok'nathal in WoW, they should be quite literally half-orc, half-ogres.

When you say Mok'nathal in WoW, do you mean Rexxar? Won't it create a problem where Rexxar will look like a different race?

Reply:
No, because the ethnicity can still generate characters that resemble Rexxar. What I meant when I said this is that there should now be more unique-looking Mok'nathal generations since otherwise they would be only based on Rexxar and Leoroxx, who are the only two Mok'nathal there's official artwork of.

However, if we run into problems with this method later on when we make Rexxar's DNA, we can come back to the Mok'nathal ethnicity and adjust the ranges again in case Rexxar's DNA falls outside of them.